templates/containers/what-are-containers.html

```diff
@@ -14,282 +14,250 @@
+    <hr class="p-rule" />
+    <div class="row--50-50 p-section--shallow">
+      <div class="col">
+        <h2>Kubernetes vs. Docker: CRI-O and OCI</h2>
+      </div>
+      <div class="col">
         <p>
-          Not clear on how Kubernetes vs Docker differ?
-          <br />
-          <a href="/blog/kubernetes-versus-docker">Read the Kubernetes vs Docker blog&nbsp;&rsaquo;</a>
+          With more and more applications deployed on containers, simple orchestration tools are not enough to support complex production environments. <a href="https://www.ubuntu.com/kubernetes/what-is-kubernetes#container-orchestration">"Container orchestration″</a> becomes a problem space of its own. The rise of <a href="https://ubuntu.com/kubernetes">Kubernetes</a> is the industry’s answer to the container orchestration problem, providing a way to manage containers at scale efficiently.
+        </p>
+        <p>
+          CRI (Container Runtime Interface) is a Kubernetes API to interface between Kubernetes orchestration features and container runtimes, such as containerd or CRI-O. The <a href="https://opencontainers.org/">Open Container Initiative</a> (OCI) defines a set of specifications to enable all these technologies to work together. Docker, runc, containerd, and CRI-O are all OCI compatible.
         </p>
-      </div>
-      <div class="col-5 u-align--center u-vertically-center u-hide--medium u-hide--small">
-        {{ image(url="https://assets.ubuntu.com/v1/cd384737-Kubernetes+v+Docker.svg",
-                alt="",
-                width="300",
-                height="105",
-                hi_def=True,
-                loading="lazy") | safe
-        }}
       </div>
     </div>
   </section>
 
-  <section class="p-strip u-no-padding--top u-no-padding--bottom">
-    <div class="p-strip--suru-topped">
-      <div class="row u-equal-height">
-        <div class="col-7 u-vertically-center">
-          <p>
-            Choosing the right Kubernetes distribution is crucial for your business.
-            <br />
-            <a href="/engage/enterprise-kubernetes-comparison">Read the Kubernetes distribution comparison whitepaper&nbsp;&rsaquo;</a>
-          </p>
-        </div>
-        <div class="col-5 u-align--center u-vertically-center u-hide--medium u-hide--small">
-          {{ image(url="https://assets.ubuntu.com/v1/198b738b-Compare_Kubernetes-Openshift-Rancher.svg",
-                    alt="",
-                    width="240",
-                    height="150",
-                    hi_def=True,
-                    loading="lazy") | safe
-          }}
+  <section class="p-section u-fixed-width">
+    <hr class="p-rule" />
+    <div class="u-fixed-width">
+      <div class="p-logo-section">
+        <div class="p-logo-section__items">
+          <div class="p-logo-section__item">
+            <img class="p-logo-section__logo"
+                 src="https://assets.ubuntu.com/v1/59fd3874-k8s-docker.png"
+                 alt="" />
```

Comment:
```suggestion
                 alt="Kubernetes and Docker logos" />
```
Logos need alt text for accessibility

Reply:
I added them previously but Juan [suggested](https://github.com/canonical/ubuntu.com/pull/13931#issuecomment-2161080370) me to remove them